scripts/Tachidesk-Server-x64.wxs

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Product Id="*" UpgradeCode="*"
+    Version="0.6.0" Language="1033" Name="Tachidesk Server" Manufacturer="Suwayomi">
```

Comment:
do we have to manually update this with each release?

maybe do the windows 4 part version number here? `0.6.0.1033`

Reply:
forgot this one. I should be able to automate it

Reply:
4 part versioning is not necessary. We could do it if you want